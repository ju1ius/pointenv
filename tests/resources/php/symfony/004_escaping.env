FOO=foo

A=a\b
B=a\\b
C='a\b'
D='a\\b'
E="a\b"
F="a\\b"

G=a\$FOO
H="a\$FOO"
I='a\$FOO'
J=a\\$FOO
K="a\\$FOO"
L='a\\$FOO'

M="1\
2"