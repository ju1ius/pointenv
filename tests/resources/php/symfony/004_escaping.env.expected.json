{
    "FOO": "foo",
    "A": "a\\b",
    "B": "a\\b",
    "C": "a\\b",
    "D": "a\\\\b",
    "E": "a\\b",
    "F": "a\\b",
    "G": "a$FOO",
    "H": "a$FOO",
    "I": "a\\$FOO",
    "J": "a\\foo",
    "K": "a\\foo",
    "L": "a\\\\$FOO"
}